geo/map_processing/map_test.py

```diff
@@ -0,0 +1,55 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''Tests for map_structure.py'''
+
+import map_structure
+import osmnx as ox
+import unittest
+from shapely.geometry.point import Point
+from map_structure import Map
+from s2geometry import pywraps2 as s2
+
+
+class MapTest(unittest.TestCase):
+
+    def setUp(self):
+
+        # Process the map for an area in Pittsburgh.
+        self.pittsburgh_map = Map("Pittsburgh")
+
+    def testSingleOutput(self):
+        # Check a known POI is there.
+        self.assertEqual(self.pittsburgh_map.poi[self.pittsburgh_map.poi[
+            'name'] == 'Frick Building'].shape[0], 1)
```

Comment:
Break this up into several lines, with variable names that make it easy to understand what is being done and checked.

Reply:
done